proposed/http-message.md

```diff
@@ -1267,12 +1271,11 @@ interface UriInterface
     /**
      * Return an instance with the specified scheme.
      *
-     * This method MUST retain the state of the current instance, and return
-     * an instance that contains the specified scheme. If the scheme
-     * provided includes the "://" delimiter, it MUST be removed.
+     * This method MUST retain the state of the current instance, and return a
+     * new instance that contains the specified scheme.
      *
      * Implementations SHOULD restrict values to "http", "https", or an empty
```

Comment:
I think this restriction is pointless. If it's about validation then it's useless since one cannot rely on it as it may accept other schemes as well.
So I would reverse the logic: "Implementations MUST support schemes  "http", "https", and an empty string but MAY accept other schemes if required."